package.json

```diff
@@ -2,17 +2,27 @@
   "name": "happy_heron",
   "private": true,
   "dependencies": {
+    "@babel/preset-typescript": "^7.12.0",
     "@rails/actioncable": "^6.0.0",
-    "@rails/activestorage": "^6.0.0",
-    "@rails/ujs": "^6.0.0",
+    "@rails/activestorage": "^6.0.3-4",
+    "@rails/ujs": "^6.0.3-4",
     "@rails/webpacker": "5.2.1",
     "bootstrap": "^5.0.0-alpha1",
     "dropzone": "^5.7.2",
     "stimulus": "^1.1.1",
-    "turbolinks": "^5.2.0"
+    "turbolinks": "^5.2.0",
+    "typescript": "^4.0.3"
```

Comment:
Should typescript be a devDependency?

Reply:
In short, I don't know. This process was not terribly well documented, and I followed the guides I found, which led to this. I leave this PR here for someone else to take forward, or not.

Reply:
Hmm, maybe it doesn't matter: https://github.com/facebook/create-react-app/issues/6180#issuecomment-453640473